frontend/src/libs/components/components.ts

```diff
@@ -1,5 +1,6 @@
 export { App } from './app/app.js';
 export { Button } from './button/button.js';
+export { Header } from './header/header.js';
```

Comment:
add Header for layout

Reply:
About that, I guess I made a header component and its styles only for the ‘my sites’ pages because of your comment. However, we have not implemented logic for the ‘my sites’ pages yet. So I am wondering where I can add my Header component to the layout. Also, in the issue I was assigned to, there is an attached picture of a header with different styles from Figma. I see that we are going to have three types of headers with different logo sizes, padding, header heights, and background colors. Do I need to make my header component have all types of styles and use a type prop to define what styles exactly I need? And should I provide other content such as profile picture, profile name, different buttons etc. as children elements? Or I should only fix your requested changes and leave like it is? 
Example: 
`type Properties = {
  type?: 'headerTransparent' | 'headerWide' | 'headerNarrow';
} & React.PropsWithChildren;

const Header: React.FC<Properties> = ({
  type = 'headerTransparent',
  children,
}: Properties => {
  return (
    <header className={getValidClassNames(styles.header, styles[type])}>
      <div className={styles.logoWrapper}>
        <div className={styles.logoIcon} />
        <span className={styles.logoText}>logo</span>
      </div>
      <div className={styles.headerSidebar}>
        { children }
      </div>
    </header>
  );
};
`  
So that gonna be like that: 
![headers](https://user-images.githubusercontent.com/87016333/225886656-6bf2df34-b3b7-4a89-8122-12dd2de95b49.png)

Reply:
You need to implement by this design:

![image](https://user-images.githubusercontent.com/42296182/225899546-0dafe058-b44e-4345-a20a-8c1672d46126.png)

and add it to the `Layout` component